### ISSUE TITLE
Trend arrow not visible in Dark UI

### ISSUE THREAD

**Issue by jpegjames** [jpegjames/glukey#1]
<img width="492" alt="screen shot 2018-02-05 at 11 08 42 pm" src="https://user-images.githubusercontent.com/117704/35857977-5e6c48e8-0b00-11e8-9b4a-9d89927a0686.png">
<img width="512" alt="screen shot 2018-02-05 at 11 07 52 pm" src="https://user-images.githubusercontent.com/117704/35857978-5e7d1362-0b00-11e8-8db8-42e5a4a78216.png">

**Comment by jpegjames** [comment]
This should work, but causes other issues

```Swift
// true should allow the arrow to invert but does not but it negatively affects the Glukey icon
iconBase?.isTemplate = true 
```

**Comment by jpegjames** [comment]
Possibly this could change to include a toggle for light or dark icon:

```Swift
self.statusItem.image = NSImage(named: "\(Constants.statusIconBase)\(current_data["Trend"]!)")
```

**Comment by jpegjames** [comment]
Fixed with cd19e3010a900304fdea5ad849b665898a0a53d1
